x-pack/plugins/alerting/server/saved_objects/model_versions/rule_model_versions.ts

```diff
@@ -30,4 +35,40 @@ export const ruleModelVersions: SavedObjectsModelVersionMap = {
       create: rawRuleSchemaV3,
     },
   },
+  '4': {
+    changes: [
+      {
+        type: 'mappings_addition',
+        addedMappings: {
+          monitoring: {
+            properties: {
+              run: {
+                properties: {
+                  last_run: {
+                    properties: {
+                      metrics: {
+                        properties: {
+                          unfilled_gaps_ms: {
+                            properties: {
+                              last_1d: { type: 'long' },
+                              last_3d: { type: 'long' },
+                              last_7d: { type: 'long' },
```

Comment:
I may have missed it, but are we searching on these fields somewhere? If not we should exclude the mappings additions.

Reply:
In the rule monitoring table we plan to use this field for sorting, and maybe filtering search in the future.